src/App.css

```diff
@@ -1,33 +1,95 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
 .App {
-  text-align: center;
+  max-width: 100%;
+  display: grid;
+  grid-template-rows: 50px 1fr;
+  grid-template-columns: 1fr;
+  width: 100vw;
+  height: 100vh;
+}
+
+.header {
+  display: flex;
```

Comment:
Why is this display: flex ? Header contains only one element.

Reply:
Probably to center the h1 and add the +Button later :)